src/main/java/core/basesyntax/Storage.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class Storage {
+    private static Map<String, TreeMap<LocalDate, Integer>> fruitsInStore = new HashMap<>();
+
+    public static TreeMap<LocalDate, Integer> getFruit(String fruit) {
+        return fruitsInStore.get(fruit);
+    }
+
+    public static void setFruit(String fruitType, LocalDate date,
```

Comment:
good, but it's better to pass a fruit directly
```suggestion
    public static void setFruit(Fruit fruit) {
```
Ok, let's leave it as is.
We will see what is DAO in the next lessons. Keep in mind this comment for a future

Reply:
done